packages/dataviews/src/filter-summary.js

```diff
@@ -62,127 +55,126 @@ const FilterText = ( { activeElement, filterInView, filter } ) => {
 	);
 };
 
-function WithSeparators( { children } ) {
-	return Children.toArray( children )
-		.filter( Boolean )
-		.map( ( child, i ) => (
-			<Fragment key={ i }>
-				{ i > 0 && <DropdownMenuSeparator /> }
-				{ child }
-			</Fragment>
-		) );
+function OperatorSelector( { filter, view, onChangeView } ) {
+	const operatorOptions = filter.operators?.map( ( operator ) => ( {
+		value: operator,
+		label: OPERATORS[ operator ]?.label,
+	} ) );
+	const currentFilter = view.filters.find(
+		( _filter ) => _filter.field === filter.field
+	);
+	const value = currentFilter?.operator || filter.operators[ 0 ];
+	return (
+		<HStack
+			spacing={ 3 }
+			justify="flex-start"
+			className="dataviews-filter-summary__operators-container"
+		>
+			{ operatorOptions.length > 1 && (
+				<SelectControl
+					label={ filter.name }
```

Comment:
@jameskoster I don't think that's semantically correct. The Select control should have a label that corresponds to its values, so the previous label(`condition` ) is the one we should use.

Reply:
Hmm good point. Ideally the appearance is:

> Author: Is

I suppose we could achieve that by using `filter.name` as the `prefix` on `SelectControl`, then visually hiding the label. What do you think?

Reply:
The prefix is rendered inside the input control, which means it's inside the borders.  Since we cannot have border-less controls(I think from other discussions recently for a11y reasons), how do you think we should move forward?

Reply:
I restored the 'conditions' label and left the name outside. Also made it so that the operators container (including the name) is hidden when there are no operators which cleans up the popover a bit.